nms/packages/magmalte/app/components/lte/LteSections.js

```diff
@@ -51,12 +51,12 @@ export function getLteSections(
   const sections = [
     'map', // landing path
     [
-      {
-        path: 'map',
-        label: 'Map',
-        icon: <PublicIcon />,
-        component: Insights,
-      },
+      // {
```

Comment:
I'm not sure what this is about. We don't support the version 1 dashboard anymore?